client/web/src/api.ts

```diff
@@ -77,3 +77,36 @@ export function setSynoToken(token?: string) {
 export function setUnraidCsrfToken(token?: string) {
   unraidCsrfToken = token
 }
+
+// postMetrics hits the client metrics local API endpoint to
+// update prometheus metrics for the node using the passed in
+// metrics data.
+export function postMetrics(d: MetricsPOSTData[]) {
+  apiFetch("/local/v0/upload-client-metrics", "POST", d)
+  .catch((error) => {
+    console.error(error)
+  })
+}
+
+
+// incrementMetric wraps postMetrics and increments the given
+// counter metric by one.
+export function incrementMetric(metricName: MetricName) {
```

Comment:
Since increment is the only possible update, you could combine this and the above function

Reply:
If not, maybe just export `incrementMetric` so that it's not confusing which one to use

Reply:
Ah good call, will adjust this.